_data/parammittal16/parammittal16.md

```diff
@@ -0,0 +1,7 @@
+---
+name: PARAM MITTAL
+institution: AJAY KUMAR GARG ENGINEERING COLLEGE
+profile_pic: parammittal16.png
```

Comment:
Hi,
Your profile pic file in the repository is parammittal16.jpeg, while its parammittal16.png in the markdown. Please fix!

Reply:
Done

Reply:
Thanks @ashwindasr

Reply:
Looks good!